test/Vagrantfile

```diff
@@ -5,7 +5,7 @@ $BUILD_NUMBER = ENV['BUILD_NUMBER'] || "0"
 $JOB_NAME = ENV['JOB_BASE_NAME'] || "LOCAL"
 $K8S_VERSION = ENV['K8S_VERSION'] || "1.7"
 $K8S_NODES = (ENV['K8S_NODES'] || "2").to_i
-
+$CILIUM_USE_ENVOY = (ENV['CILIUM_USE_ENVOY'] || "0")
```

Comment:
Defining CILIUM_USE_ENVOY as "0" does not have the desired effect as envoy/Makefile tests if CILIUM_USE_ENVOY is defined and ignores the value.

Also, to reflect the main Vagrantfile, IMO test/Vagrantfile should test or even read the CILIUM_USE_ENVOY environment variable, but unconditionally define CILIUM_USE_ENVOY=1 for the provisioning scripts.

Once we have everything successfully moved over to Envoy, we'll be removing "CILIUM_USE_ENVOY" env variable altogether.